bin/make_db

```diff
@@ -1,31 +1,18 @@
+#!/usr/bin/env python3
+"""Initialize the dev environment's DB."""
```

Comment:
h's new `bin/make_db` script is similar to [the one in the cookiecutter](https://github.com/hypothesis/cookiecutters/blob/main/pyramid-app/%7B%7B%20cookiecutter.slug%20%7D%7D/bin/make_db) but not exactly the same. When applying the cookiecutter to h it'll simply override this whole `make_db` file and supply its own.

I've left the SQLAlchemy code in here the same as it was in h's existing `h/cli/commands/init.py` file although I could probably rewrite it to be the same as the functionally equivalent code in the cookiecutter's `make_db` and I think that'd work fine. (But h's `make_db` would still be a bit different because it initializes Elasticsearch as well.)